diesel/src/pg/connection/mod.rs

```diff
@@ -26,6 +26,7 @@ use types::{ToSql, HasSqlType};
 /// The connection string expected by `PgConnection::establish`
 /// should be a PostgreSQL connection string, as documented at
 /// http://www.postgresql.org/docs/9.4/static/libpq-connect.html#LIBPQ-CONNSTRING
+#[allow(missing_debug_implementations, missing_copy_implementations)]
```

Comment:
I don't think the `missing_copy_implementation` needs to be explicitly allowed here, does it?